templates/pages/account/edit-payment-method.html

```diff
@@ -7,48 +7,68 @@ <h2 class="page-heading">{{lang 'forms.payment_methods.edit.heading'}}</h2>
+                    {{#if type '===' 'stored_card'}}
+                        <dl class="paymentMethodForm-details">
+                            <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.credit_card_number'}}</dt>
+                            <dd class="paymentMethodForm-details-description">**** **** **** {{last_4}}</dd>
+                        </dl>
+                        <dl class="paymentMethodForm-details">
+                            <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.expiration'}}</dt>
+                            <dd class="paymentMethodForm-details-description">{{expiry_month}}/{{expiry_year}}</dd>
+                        </dl>
+                    {{/if}}
+                    {{#if type '===' 'stored_paypal_account'}}
+                        <div class="form-field">
+                            <h3 class="paymentMethodForm-heading">
+                                {{lang 'account.payment_methods.paypal'}} {{lang 'common.account'}}
+                            </h3>
+                            <p>
+                                {{email}}
+                            </p>
+                        </div>
+                    {{/if}}
+                    <div class="form-field">
+                        <input type="checkbox" name="is_default" id="default_instrument" data-label="{{lang 'forms.payment_methods.default_instrument'}}" class="form-checkbox" {{#if is_default}}checked{{/if}}>
+                        <label class="form-label" for="default_instrument">{{lang 'forms.payment_methods.default_instrument'}}</label>
+                    </div>
                 </div>
-            </fieldset>
-            <div class="form-actions">
-                <input type="submit" class="button button--primary" value="{{lang 'forms.payment_methods.submit_value'}}">
-                <a href="{{urls.account.payment_methods.all}}" class="button">{{lang 'common.cancel'}}</a>
-                {{inject 'required' (lang 'common.required')}}
-                {{inject 'state_error' (lang 'errors.state_error')}}
-            </div>
-        </form>
+
+                {{#if type '===' 'stored_card'}}
+                    <h3 class="paymentMethodForm-heading">{{lang 'account.payment_methods.billing_address'}}</h3>
+
+                    <fieldset class="form-fieldset">
+                        <div class="form-row form-row--half">
+                            {{#each forms.billing_fields}}
+                                {{{dynamicComponent 'components/common/forms'}}}
+                            {{/each}}
+                        </div>
+                    </fieldset>
+                    <div class="form-actions">
+                        <input type="submit" class="button button--primary" value="{{lang 'forms.payment_methods.submit_value'}}">
+                        <a href="{{urls.account.payment_methods.all}}" class="button">{{lang 'common.cancel'}}</a>
```

Comment:
Please change to `href="{{../urls.account.payment_methods.all}}"` as `urls.account.payment_methods.all` is not within `edit_stored_instrument` scope, otherwise the Cancel button is not going back to Payment Methods page.

Reply:
fixed